detekt/defs.bzl

```diff
@@ -2,6 +2,69 @@
 Rule declarations.
 """
 
+_ATTRS = {
```

Comment:
Little hard to tell from the Github diff, is this just a copy/paste of the original attributes or is there anything specific in here that we need to review?

Reply:
no changes in the attributes, just a shuffle (copy-and-paste).

The only attribute change comes within the `get_default_attributes` function: https://github.com/davidkurkov/bazel_rules_detekt/pull/1/files#diff-d7c07eade8964ea97850c37c83b90f69b8b3058475560766b6d40f13aa7a8350R61-R66